app/v1/controllers/requestParsers/validators/SubmitValidator.scala

```diff
@@ -0,0 +1,82 @@
+    List(
+      VrnValidation.validate(data.vrn)
+    )
+  }
+
+  private def monetaryFormatValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    val body: SubmitRequestBody = data.body.json.as[SubmitRequestBody]
+
+    List(
+      MonetaryValueFormatValidation.validate(body.vatDueSales, "vatDueSales"),
+      MonetaryValueFormatValidation.validate(body.vatDueAcquisitions, "vatDueAcquisitions"),
+      MonetaryValueFormatValidation.validate(body.totalVatDue, "totalVatDue"),
+      MonetaryValueFormatValidation.validate(body.vatReclaimedCurrPeriod, "vatReclaimedCurrPeriod"),
+      MonetaryValueFormatValidation.validate(body.netVatDue, "netVatDue"),
+      MonetaryValueFormatValidation.validate(body.totalValueSalesExVAT, "totalValueSalesExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalValuePurchasesExVAT, "totalValuePurchasesExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalValueGoodsSuppliedExVAT, "totalValueGoodsSuppliedExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalAcquisitionsExVAT, "totalAcquisitionsExVAT"),
+    )
+  }
+
+  private def monetaryValueValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    val minNetVatDue = BigDecimal(0.00)
+    val maxNetVatDue = BigDecimal(99999999999.99)
+    val minRegularValue = BigDecimal(-9999999999999.99)
+    val maxRegularValue = BigDecimal(9999999999999.99)
+
+    val body: SubmitRequestBody = data.body.json.as[SubmitRequestBody]
+
+    List(
+      MonetaryValueRangeValidation.validate(body.vatDueSales, "vatDueSales", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.vatDueAcquisitions, "vatDueAcquisitions", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalVatDue, "totalVatDue", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.vatReclaimedCurrPeriod, "vatReclaimedCurrPeriod", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.netVatDue, "netVatDue", minNetVatDue, maxNetVatDue),
+      MonetaryValueRangeValidation.validate(body.totalValueSalesExVAT, "totalValueSalesExVAT", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalValuePurchasesExVAT, "totalValuePurchasesExVAT", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalValueGoodsSuppliedExVAT, "totalValueGoodsSuppliedExVAT", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalAcquisitionsExVAT, "totalAcquisitionsExVAT", minRegularValue, maxRegularValue),
+
+    )
+  }
+
+  private def periodKeyFormatValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    val body: SubmitRequestBody = data.body.json.as[SubmitRequestBody]
+
+    List(
+      PeriodKeyValidation.validate(body.periodKey)
```

Comment:
This period key validator uses an error with different wording to the error thrown on this endpoint.